modules/ROOT/pages/traversal-framework/index.adoc

```diff
@@ -0,0 +1,57 @@
+:description: The Neo4j Traversal Framework Java API.
+
+:org-neo4j-graphdb-Direction-both: {neo4j-javadocs-base-uri}/org/neo4j/graphdb/Direction.html#BOTH
+
+
+[[traversal]]
+= The Traversal Framework
+
+This section provides an overview of the Traversal Framework, and a detailed description of the Neo4j Traversal Framework Java API and how to use it.
+
+The Neo4j Traversal Framework Java API is a callback-based, lazily-executed way of specifying desired movements through a graph in Java.
```

Comment:
Is "lazily" a technical word here or can we find some synonym that feels less colloquial? For example, "simplified" or "straightforward"?

Reply:
I think that Lazy is OK here but maybe more explanation could be good, eg it's a Stream